src/spider/storage/mysql/MySqlStorage.cpp

```diff
@@ -256,13 +281,16 @@ void MySqlMetadataStorage::add_task(
     task_statement->setBytes(1, &task_id_bytes);
     task_statement->setBytes(2, &job_id);
     task_statement->setString(3, task.get_function_name());
+    auto const task_language
+            = YSTDLIB_ERROR_HANDLING_TRYX(task_language_to_string(task.get_language()));
+    task_statement->setString(4, task_language);
```

Comment:
```suggestion
    task_statement->setString(
            4,
            YSTDLIB_ERROR_HANDLING_TRYX(task_language_to_string(task.get_language()))
    );
```
Let's put a rvalue directly, just in case later mariadb API changes will introduce more efficient constructors (like move).